src/prefect/server/models/deployments.py

```diff
@@ -1052,58 +1056,60 @@ async def delete_deployment_schedule(
     return result.rowcount > 0
 
 
-@db_injector
 async def mark_deployments_ready(
     db: PrefectDBInterface,
     deployment_ids: Optional[Iterable[UUID]] = None,
     work_queue_ids: Optional[Iterable[UUID]] = None,
 ) -> None:
-    deployment_ids = deployment_ids or []
-    work_queue_ids = work_queue_ids or []
+    try:
```

Comment:
i think you may still need to include the `db_injector` logic here

Reply:
hm tests still pass

Reply:
hmm my impression was that db_injector was just a really complicated and type-safe way to inject `db = provide_database_interface()` into the signature

Reply:
yea, this may be a dumb question but why don't you need to call `db = provide_database_interface()` in any of these functions that you removed the `db_injector` from?

Reply:
bc im not explicitly passing it in, its only used in places where its already a `Depends` in the endpoint signature